source/core/assets/configs/NPCs.json

```diff
@@ -7,7 +7,10 @@
     "animalName": "Lion"
   },
   "cow": {
-    "baseHint": ["Welcome to Animal Kingdom!", "I am Charlie the Cow."],
+    "baseHint": ["Moo there, adventurer! Welcome to the kingdom.",
+      "We’ll be your guides",
```

Comment:
Issue with approstrophie showing on dialogue box, change to either "we will" or investigate the error